core/load.py

```diff
@@ -884,6 +887,9 @@ def load_honeypot_engine():
     )
     # killed the network traffic capture process by ctrl + c... waiting to end.
     info(messages["killing_capture_process"])
+    if exit_flag:
+        # Terminate the network capture process
+        network_traffic_capture_process.terminate()
```

Comment:
this should do the job right without the above changes?

Reply:
nopes, we need a exit flag value which denotes this has exited